pkgs/development/libraries/audio/roc-toolkit/default.nix

```
@@ -37,15 +39,16 @@ stdenv.mkDerivation rec {
     libunwind
     openfec
     libpulseaudio
+    sox
   ];
```

While we're at this:

```suggestion
  ]
  ++ lib.optional libunwindSupport libunwind
  ++ lib.optional openfecSupport openfec
  ++ lib.optional pulseaudioSupport libpulseaudio
  ++ lib.optional soxSupport sox;
```

---

That's why I didn't merge this when I last looked at it, but was going to add it when it was brought up again a few days ago.

Anyways, I don't think it's worth breaking compatibility for options that don't really affect us at all, providing more customization is always good.

I'll push that change as soon as I can and land this.